_docs_integrate/iql-syntax.md

```diff
@@ -27,13 +27,19 @@ The enmeshed _Identity Query Language_ (IQL) is a domain-specific language to qu
 
 ### Filter by Value Type
 
-The most common usage of the IQL is to filter [IdentityAttributes]({% link _docs_integrate/attribute-values.md %}#identity-attributes) by their value types or tags. Value types are specified by directly using their names as terms in the query string. For example, the following IQLQuery will match all phone numbers, i.e. all IdentityAttributes of value type _PhoneNumber_:
+The most common usage of the IQL is to filter [IdentityAttributes]({% link _docs_integrate/attribute-values.md %}#identity-attributes) by their [value types]({% link _docs_integrate/attribute-values.md %}#identity-attributes) or tags. Value types are specified by directly using their names as terms in the query string. For example, the following IQLQuery will match all phone numbers, i.e. all IdentityAttributes of value type _PhoneNumber_:
```

Comment:
Two links to the IdentityAttribute value types.  Link the first time to the [IdentityAttributes]({% link _docs_integrate/data-model-overview.md %}#identityattribute) in the data model overview instead.